### ISSUE TITLE
[PNI Privacy Index] Implementation Epic

### ISSUE THREAD

**Issue by mmmavis** [MozillaFoundation/foundation.mozilla.org#11047]
Placeholder ticket. More to come once design have been finalized.

--- 

- [Project PRD](https://docs.google.com/document/d/1tydN39iahB4RUwWZPgZazdY1b11dDoRiLDRX5Zp150g/edit)

**Comment by Gbaldassano** [comment]
@mmmavis flagging a note about timing for launch here: 

- Launch the site on Oct. 16 with password for press embargo
- Publicly unveil it Oct. 18 (remove password)

@mtdenton since Mavis will be OOO on 10/16, can we launch the page with the password the Thursday (10/12) before?

**Comment by mtdenton** [comment]
Yes, assuming everything is in place, we can launch on 10/12. We need to make sure that we keep things light on `main` for 10/10 and 10/11 just in case
